docs/api/tenacity.md

```diff
@@ -0,0 +1,23 @@
+# `pydantic_ai.tenacity`
+
+## Setup
+
+To use the tenacity integration, you need to install the `tenacity` dependency:
+
+```bash
+pip install tenacity
+```
+
+## Overview
```

Comment:
Do we need this doc? I'd prefer to just refer to `../retries.md` and do everything there